src/sentry/integrations/slack/message_builder/base/block.py

```diff
@@ -120,10 +120,11 @@ def _build_blocks(
         fallback_text: Optional[str] = None,
         color: Optional[str] = None,
         block_id: Optional[dict[str, int]] = None,
+        skip_fallback: bool = False,
     ) -> SlackBlock:
         blocks: dict[str, Any] = {"blocks": list(args)}
 
-        if fallback_text:
+        if fallback_text and not skip_fallback:
```

Comment:
I had to add this for the backwards compatibility for the resolve modal. I can't explain why, but I always got an "invalid attachment" error message when adding the fallback text for this specifically. I don't have this issue with the assign or ignore buttons with backwards compatibility. It will not have the popup text but I think it's okay since it's just a temporary holdover. See [Slack docs](https://api.slack.com/reference/messaging/payload) on this - I see no reason why it should fail, but it might be some bug specific to after closing a modal.

Reply:
I tried this and unfortunately I still get the `invalid_attachments` error :(